hw4/css/style-hw4.css

```diff
@@ -0,0 +1,238 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+	background: linear-gradient( to top, #0c739f 0%, #40c5d2 80%, #0c739f 100%);
+	min-width: 500px;
+	color: #000;
+	font-family: Tahoma, Arial, Helvetica, sans-serif;
+}
+
+.background {
```

Comment:
you know I have just realized good way to make things like in design. What if we would use this background conatiner with it's borders? I mean create the center part with background and bottom and top partes as a borders?